kolibri/core/auth/management/commands/bulkimportusers.py

```diff
@@ -579,6 +579,21 @@ def build_users_objects(self, users):
                 if self.compare_fields(user_obj, values):
                     update_users.append(user_obj)
             else:
+                # If UUID is not specified, check for a username clash
+                if values["uuid"] == "":
+                    existing_user = FacilityUser.objects.filter(
+                        username=user, facility=self.default_facility
```

Comment:
We should do a case insensitive match here as well, to parallel the changes you made above.